app/validators/state_validator.rb

```diff
@@ -0,0 +1,12 @@
+class StateValidator < ActiveModel::Validator
+  def validate(record)
```

Comment:
Can we use an inclusivity validator for this? The error messages tend to be pretty. Something similar to how ViewingHintValidator works.